narrbase-image/Dockerfile

```diff
@@ -18,13 +20,20 @@ RUN conda update -n base conda && \
     conda install -c conda-forge --file /root/conda/r.conda-forge && \
     conda install -c r --file /root/conda/r.r
 
+# Install apt-get prereqs for node, R
+RUN apt-get update && \
+    apt-get install -y gfortran gnupg
+
+# Install nodejs at a useful version
+RUN curl -sL https://deb.nodesource.com/setup_${NODEJS_VERSION}.x | bash - && \
+    apt-get install -y nodejs
+
 # Install misc R packages not available on Coda
```

Comment:
Conda

Reply:
Done. conda-forge has the latest nodejs LTS, so that should be ok.

Reply:
😆 I'm totally in-favor of this change but I actually was just talking about the spelling of the comment "Install misc R packages not available on *Coda*"

Reply:
Ha. Oops! Fixed it. 😝